content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,48 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind PVCs to matching PVs](#binding) in the
+cluster. However, if you want a PVC to bind to a specific PV, you need to
+pre-bind them.
+
+You can specify the PV in your PVC, then the PVC will be able to bind to this
+specific PV. If the PV exists and is not reserved for other PVCs (has PVC
+specified in `claimRef` field), the PV and PVC will be bound regardless of
+whether the PV satisfies some matching criteria, e.g. node affinity. Storage
+class, access modes, and requested storage size are still checked.
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: foo-pvc
+  namespace: foo
+spec:
+  volumeName: foo-pv
+  ...
+```
+
+Note that this method does not guarantee the PV right. If the PV you specified
+may be used by other PVCs, you need to reserve it first by specifying the PVC
+in the PV using the `claimRef` field so that other PVCs can not bind to it.
```

Comment:
```suggestion
may be used by other persistent volume claims, you need to reserve it first by specifying the persistent volume claim
in the persistent vlume using the `claimRef` field so that other persistent volume claims can not bind to it.
```